subsys/bluetooth/audio/Kconfig.ascs

```diff
@@ -7,9 +7,7 @@
 #
 
 config BT_ASCS
-	bool "Audio Stream Control Service Support"
-	help
-	  This option enables support for Audio Stream Control Service.
```

Comment:
This can be argued to be fine for now. 

The reason why it was done the other way around, is that the `Unicast Server` is a BAP role, and this is a GATT service. 

What this means is that not only is it possible that other BT SIG profiles may use ASCS, but ASCS can even be used outside any BT SIG defined roles. 

However, since the current implementation is heavily coupled with the BAP Unicast Server role, this change is fine, but less generic/flexible, but we can also "fix" that if/when ASCS would ever be used in other contexts than the BAP Unicast Server role.